src/patternfly/components/LabelGroup/label-group.scss

```diff
@@ -0,0 +1,98 @@
+.pf-c-label-group {
```

Comment:
should there be truncation when the labels are too long ? cc: @mcarrano @mceledonia 
<img width="799" alt="Screen Shot 2020-08-18 at 1 21 07 PM" src="https://user-images.githubusercontent.com/20118816/90544847-bb5b1600-e155-11ea-85c6-bc1a60657468.png">

Reply:
@christiemolloy we added support for that after the fact, via a `pf-c-label__text` element, so that can be added to the chips. There are examples of it on https://www.patternfly.org/v4/documentation/core/components/label (the last label in each row). There is an open react issue (https://github.com/patternfly/patternfly-react/issues/4660) to add support for that via a prop, and in a breaking change release, we'll make that the default, which will match the chip. I added an issue for that update here - https://github.com/patternfly/patternfly/issues/3409